src/index.js

```diff
@@ -1,5 +1,24 @@
-let {assoc, assocPath, append, contains, curry, drop, dropWhile, identity, join, keys} = require("ramda")
-let {map, merge, pipe, prepend, split, reduce, repeat, replace, takeWhile, trim} = require("ramda")
+let assoc = require("ramda/src/assoc");
```

Comment:
Can we replace this list with `R`? I started to use this namespace in my later projects and while I was super negative about it before – I freaking like it today.
It was just a matter of habit (as most other things).

`R` for Ramda
`P` for Path
`L` for Lenses